### ISSUE TITLE
Fix broken links

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/standard#552]
Originally reported in comment: https://github.com/open-contracting/standard/issues/521#issuecomment-322112082

The following links are broken in `levels.md`:

```
[a JSON schema](../../../schema/reference/)
```

and

```
[established protocols](../../hosting#discovery-and-apis)
```

**Comment by Bjwebb** [comment]
I've written a test to catch broken links, at @duncandewhurst's suggestion.

There's a lot of them, to the point where I think something has changed in how Sphinx handles relative links. I've also noticed that the behaviour of links in the English docs seems to be different to translations https://github.com/open-contracting/standard/pull/556/files#r136608618. I think some poking around with our Sphinx setup (e.g. updating versions) could be useful before going through and trying to fix these.

Page|Link destination
---|---
/en/getting_started/building_blocks/|/en/getting_started/extensions
/en/getting_started/publication_patterns/|/en/getting_started/schema/release_package
/en/getting_started/publication_patterns/|/en/getting_started/schema/record_package
/en/schema/reference/|/en/schema/reference/codelists#release-tag
/en/schema/reference/|/en/schema/extensions/process_title
/en/schema/reference/|/en/schema/extensions/party_details
/en/schema/reference/|/en/schema/extensions/enquiries
/en/schema/reference/|/en/schema/extensions/lots
/en/schema/reference/|/en/schema/extensions/participation_fee
/en/schema/reference/|/en/schema/extensions/bids
/en/schema/reference/|/en/schema/reference/identifiers#organisation-ids
/en/schema/reference/|/en/schema/reference/codelists#document-type
/en/schema/reference/|/en/schema/extensions/location
/en/schema/reference/|/en/schema/extensions/milestone_documents
/en/schema/records_reference/|/en/schema/records_reference/identifiers#ocid
/en/schema/records_reference/|/schema/reference.html#package-metadata
/en/schema/codelists/|/en/schema/support
/en/schema/codelists/|/en/schema/codelists/reference#parties
/en/schema/conformance_and_extensions/|/en/schema/conformance_and_extensions/extensions
/en/schema/deprecation/|/en/schema/support/governance#deprecation-policy
/en/implementation/levels/|/en/implementation/implementation/serialization#csv
/en/implementation/levels/|/en/implementation/schema/reference
/en/implementation/levels/|/en/implementation/levels/hosting#discovery-and-apis
/en/implementation/licensing/|/en/implementation/getting_started/use_cases
/en/implementation/serialization/|/en/implementation/schema/release
/en/implementation/hosting/|/en/implementation/hosting/serialization
/en/implementation/registration/|/en/implementation/implementation/hosting
/en/implementation/registration/|/en/implementation/support
/en/extensions/party_details/|/en/extensions/schema/reference#parties
/en/support/governance/|/en/support/governance/support/support/mailto:data@open-contracting.org
/en/support/governance/|/en/support/governance/support/support/mailto:standard-discuss@open-contracting.org
/en/support/governance/|/en/support/governance/support/support/standard-discuss+subscribe@open-contracting.org

**Comment by Bjwebb** [comment]
I've tried reverting the most recent update to our Sphinx/recommonmark dependencies - https://github.com/open-contracting/standard/commit/aaf2b2a676aa60c153243f311871b99ab335269d - but the list of broken links is the same.

**Comment by timgdavies** [comment]
I've assigned @duncandewhurst  in case Duncan has an opportunity to look at this before travels - as would be good if we can fix those links soon.

**Comment by duncandewhurst** [comment]
@Bjwebb does that mean this is now a manual task or are you still looking at Sphinx/recommonmark fixes?

**Comment by Bjwebb** [comment]
I'm a bit stumped on Sphinx/recommonmark fixes. I think this is maybe something we'll revisit, but for now I think making the manual change makes sense.

It's probably easier for someone who has the tests setup locally to go through and fix these. I can do so on Wednesday if you want.

**Comment by duncandewhurst** [comment]
That would be great, thanks @Bjwebb - let me know if you don't get round to it and I'll pick up later in the week.

**Comment by Bjwebb** [comment]
To fix some of these I need to make a change to https://github.com/open-contracting/ocds_sphinx_directives. I don't seem to have push access to this any more, @jpmckinney could you grant me access?

**Comment by jpmckinney** [comment]
Access issue is related to https://github.com/open-contracting/standard-development-handbook/issues/12 (Previously, almost all of ODS was an 'owner' of the org.) I added that repo to the Developers team, but we should really just schedule 30 mins to configure teams properly.

**Comment by jpmckinney** [comment]
By the way, @robredpath and @timgdavies are still owners. We can swap these around, but I prefer just two owners per organization: https://github.com/open-contracting/standard-development-handbook/issues/18#issuecomment-331143065

**Comment by Bjwebb** [comment]
Thanks. @robredpath and @timgdavies probably make sense as owners, they just both happen to be away this week!

**Comment by jpmckinney** [comment]
@Bjwebb Is this issue fixed by #579 and can be closed, or is there a reason to keep it open?

**Comment by Bjwebb** [comment]
This is fixed now.
